sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefEditorServiceTest.java

```diff
@@ -4,6 +4,7 @@
 
 package org.lfenergy.compas.sct.commons;
 
+import jakarta.xml.bind.Marshaller;
```

Comment:
Unused import

Reply:
Thanks, I did not configure the "commit Optimize import' on this project.